app-scientific/sagemath/autobuild/build

```diff
@@ -0,0 +1,8 @@
+export SAGE_LOCAL=/usr/lib/sagemath/local
+make configure
+./configure --disable-editable --prefix=$SAGE_LOCAL --enable-build-as-root
+make
+mkdir -p "$PKGDIR"/usr/lib
+cp -av /usr/lib/sagemath "$PKGDIR"/usr/lib/
+mkdir -p "$PKGDIR"/usr/bin
+ln -s /usr/lib/sagemath/local/bin/sage "$PKGDIR"/usr/bin/sage
```

Comment:
Verbosity here; also, always use relative path to avoid rootfs pollution when accessing from other systems:

```
ln -sv ../lib/sagemath/local/lib/sage \
    "$PKGDIR"/usr/bin/sage
```